content/en/blog/2025/devex-survey/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Insights from the OpenTelemetry Developer Experience Survey
+linkTitle: Insights from the Developer Experience Survey
+date: 2025-03-26
+author: >
+  [Tristan Sloughter](https://github.com/tsloughter) (MyDecisiveAI), [Juliano
+  Costa](https://github.com/julianocosta89) (Datadog), [Steve
+  Gordon](https://github.com/stevejgordon) (Elastic), [Damien
+  Mathieu](https://github.com/dmathieu) (Elastic)
+sig: Developer Experience
+---
+
+# Insights from the OpenTelemetry Developer Experience Survey
+
+The OpenTelemetry Developer Experience SIG recently surveyed the community to
+better understand where the SIG could have the most impact on improving the
+developer experience. We received 218 responses, which we will use to guide our
+prioritization of work within the SIG. This post summarizes our findings and
+where we will focus in the near term.
+
+Thank you to all those who participated in the survey\! If you are interested in
```

Comment:
```suggestion
Thank you to all those who participated in the survey\! If you are interested in
```

The escaped ! is a bit strange.

Reply:
Yea, google docs exporting to markdon added those and I just left it since it worked, but works without too so I've now removed them.